automatic/lightscreen/tools/install.ahk

```diff
@@ -0,0 +1,10 @@
+#NoEnv
+#NoTrayIcon
+SendMode Input
+SetWorkingDir %A_ScriptDir%
+
+WinActivate, Options - Lightscreen
+Send, {Enter}
```

Comment:
`SendControl` should be used if possible or if everything else fails, `BlockInput`. Something can steal the focus.